ktp/src/main/kotlin/toothpick/ktp/extension/ScopeExtension.kt

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
We have to make this more uniform. Sometimes we use Extension suffix with an s and sometimes not.